app/code/Magento/Sales/Model/Order.php

```diff
@@ -387,7 +394,8 @@ public function __construct(
         OrderItemRepositoryInterface $itemRepository = null,
         SearchCriteriaBuilder $searchCriteriaBuilder = null,
         ScopeConfigInterface $scopeConfig = null,
-        RegionFactory $regionFactory = null
+        RegionFactory $regionFactory = null,
+        ?PricePrecisionInterface $pricePrecision = null
```

Comment:
I don't think it makes sense to allow to pass `null` as argument explicitly
```suggestion
        PricePrecisionInterface $pricePrecision = null
```

Reply:
Sure @sivaschenko, I thought that should be done according to **Adding a constructor parameter** point from https://devdocs.magento.com/contributor-guide/backward-compatible-development/, I can change that

Reply:
Hm, this change was introduced in https://github.com/magento/devdocs/pull/6033

I don't think that is correct.

@sidolov @gabrieldagama what do you think guys?